.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plugins"]
+	path = plugins
+	url = ../example-plugins
```

Comment:
Sorry, I might be missing something. Shouldn't this be pointing at a repo?

Reply:
The relative path is what's pointing at a repo. It's easier (and better) than using github.com/ urls.

Reply:
Sorry, I should probably have phrased that better. You can use relative paths when specifying submodules for repos in the same group. That way we don't have to hardcode https://github.com/beer-garden/example-plugins, we can just rely on `example-plugins` being a sibling of `bartender` in the `beer-garden` group.

Reply:
Ah okay cool. Thanks